### ISSUE TITLE
[FP] Unqualified function/constant reference on string references

### ISSUE THREAD

**Issue by orklah** [kalessil/phpinspectionsea#1524]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Utlimate last version |
| Language level | PHP 7.4                        |

In a namespace, the following code will trigger an "Unqualified function/constant reference".
```php
<?php

namespace a;

function intval($a){
    echo "scoped intval";
}

array_map('intval', [1]);
```
However, as mentionned here: https://github.com/Roave/SecurityAdvisoriesBuilder/pull/115#discussion_r386051823
The string already reference the global scope, as shown in the example:
https://3v4l.org/N08Yf
(Only the third call is going through the function in the namespace. The \ is redundant and the inspection shouldn't be triggered here.
